scripts/go.mod

```diff
@@ -0,0 +1,14 @@
+module scripts
+
+go 1.13
```

Comment:
What will happen if we upgrade the go version in the scripts module.

Reply:
I think there is no difference, as our CI is using Go1.13.